src/asm/main.cpp

```diff
@@ -68,6 +69,7 @@ static option const longopts[] = {
     {"gfx-chars",       required_argument, nullptr,  'g'},
     {"include",         required_argument, nullptr,  'I'},
     {"dependfile",      required_argument, nullptr,  'M'},
+    {"MC",              no_argument,       &depType, 'C'},
```

Comment:
I'd like the flag to be named something else, as this “long option with only one dash” is weird and non-standard. (It means that `-MG` and -M G` are different, which is irregular behaviour.)

E.g. `--continue-missing`, `--assume-empty`, `--keep-going`...

Reply:
Getting rid of them would be annoying for many people, so it's a no-go. Keep in mind that any breakage will push people not to upgrade. :/